fcrepo-http-api/src/main/java/org/fcrepo/http/api/FedoraLdp.java

```diff
@@ -271,6 +273,9 @@ public Response createOrReplaceObjectRdf(
 
         if (httpConfiguration.putRequiresIfMatch() && StringUtils.isBlank(ifMatch) && !resource.isNew()) {
             throw new ClientErrorException("An If-Match header is required", 428);
+        } else if (resource.hasType(FEDORA_PAIRTREE)) {
+            throw new ForbiddenResourceModificationException("Graphs cannot be created for resources " +
+                                                              "under pairtree nodes");
```

Comment:
I'm not worried about the word "graph" and if you like it better without it, fine by me. I'm confused about whether at this moment in the workflow the client is trying to create anything under `resource` or is trying to create `resource` or what. I just want to make sure a) that this test is coming at the right time in the workflow and b) that it is throwing out an exception with the right message.